components/HeaderMobile.tsx

```diff
@@ -81,120 +77,119 @@ export function HeaderMobile({
   )
 }
 
-interface HoverableItemProps {
-  className?: string
-  width: number
-  children: ReactNode
+function ProductsAccordionMenu(){
+
+  return(
+    <AccordionMenu name="Products">
+
+      <AccordionMenuItem
+        href="https://dex.polarfox.io"
+        linkTitle="Open app"
+        name="Decentralized Exchange"
+      />
+
+      <AccordionMenuItem
+        href="https://analytics.polarfox.io"
+        linkTitle="Open app"
+        name="Analytics"
+      />
+        
+      <AccordionMenuItem
+        href="https://dex.polarfox.io/#/pfx"
+        linkTitle="Open app"
+        name="Mining Pools"
+      />            
+
+      <AccordionMenuItem
+        href="https://bridge.polarfox.io"
+        linkTitle="Open app"
+        name="Bridge"
+      />
+
+      <AccordionMenuItem
+        href="https://akita.network"
+        linkTitle="Visit site"
+        name="AKITA Network"
+      />
+
+    </AccordionMenu>
+  )
+
 }
 
-function HoverableItem({ className, width, children }: HoverableItemProps) {
-  return (
-    <div
-      className={classNames(
-        'rounded-3xl flex items-center justify-between px-4 group-hover:bg-gray-mid2',
-        className
-      )}
-      style={{ width: calcRem(width), height: calcRem(44) }}
-    >
-      {children}
-      <MenuArrow />
-    </div>
+function ResourcesAccordionMenu(){
+  return(
+    <AccordionMenu name="Resources">
+
+      <AccordionMenuItem
+        href="/tutorials"
+        linkTitle="Visit site"
+        name="Tutorials"
+      />
+
+      <AccordionMenuItem
+        href="/Polarfox Litepaper V2.6.pdf"
```

Comment:
This link appears four times in the app (and it's not the only one suffering from this).

How about this: we can move all the links to a constant file, so it's easier for us to update.

(This should be done in another pull request).

Reply:
Also remove empty line